src/components/apps/Notes/stores.js

```diff
@@ -0,0 +1,18 @@
+import {writable} from 'svelte/store';
+
+let lastId = 0;
+
+// Keys are ids and values are objects with id, title, and text properties.
+export const notesStore = writable({});
```

Comment:
Would be useful to define a `Note` type here.

```ts
type Note = {
  id: number;
  title: string;
  text: string;
}

Then later define this line as

```ts
export const notesStore = writable<Record<number, Note>>({})